add MovieDB neuron to community neuron list

royto opened the pull request:
c

Commit:
add MovieDB neuron to community neuron list

Sispheor commented:
I love it ! Thanks a lot for your contribution !

Just a couple thing to update.

You need to place your synapse example into `{% raw %}` `{% endraw %}`  [like here](https://github.com/kalliope-project/kalliope-project.github.io/blob/sources/_includes/neuron_examples/Gmaps/fr_synapse.yml). Or the output is weird.

In your neuron, with the last version of kalliope you have to pass parameter to your neurons with "{{ my_parameter }}" and not anymore with "args" value in the dict. [See here for the doc](https://github.com/kalliope-project/kalliope/blob/master/Docs/neurons.md#input-values).

Commit:
integrate MR remarks

add {% raw %} and {% endraw %} ro synapse file
update synapse to use new way of passing args to neuron

Sispheor commented:
Merged, thx !!

royto commented:
Thanks !
